driver-core/src/main/com/mongodb/client/model/geojson/MultiPolygon.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.geojson;
+
+import java.util.Collections;
+import java.util.List;
+
+import static com.mongodb.assertions.Assertions.isTrueArgument;
+import static com.mongodb.assertions.Assertions.notNull;
+
+/**
+ * A representation of a GeoJSON MultiPolygon.
+ *
+ * @since 3.1
+ */
+public final class MultiPolygon extends Geometry {
+
+    private final List<PolygonCoordinates> coordinates;
```

Comment:
This is the only one, I believe.  And it's just because polygon coordinates have two parts: exterior and holes.  So this was the only way to pass a list of them.  Note that the Polygon class has a constructor that takes one of these, and another one that takes the two pieces separately.